src/sentry/conf/server.py

```diff
@@ -1666,6 +1666,10 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 )
 SENTRY_EVENT_PROCESSING_STORE_OPTIONS: dict[str, str] = {}
 
+# Transactions processing backend
+SENTRY_TRANSACTION_PROCESSING_STORE: str | None = None
+SENTRY_TRANSACTION_PROCESSING_STORE_OPTIONS: dict[str, str] | None = None
```

Comment:
nit, but one less thing to remember later
```suggestion
SENTRY_TRANSACTION_PROCESSING_STORE_OPTIONS: dict[str, str] = {}
```